features/map_test.go

```diff
@@ -0,0 +1,87 @@
+package features
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+)
+
+var testCases = map[ebpf.MapType]string{
```

Comment:
This is a (testing) package-global variable, so the name `testCases` might be a bit too generic. Maybe `mapTypeKernels`?